discover/analytics-cloud/articles/02-getting-started/02-adding-a-liferay-dxp-data-source.markdown

```diff
@@ -50,41 +50,53 @@ data source! It's time to add your Liferay DXP instance as a data source.
 Adding a Liferay DXP Data Source connects your Analytics Cloud project with
 a Liferay DXP instance. 
 
-Here's how to add a Liferay DXP data source:
+Here's how to start adding your Liferay DXP data source:
+
+1.  Select *Settings* in the navigation panel.
+
+2.  Click *Data Sources*. A listing of your existing data sources appears.
 
-1.  If you haven't already, configure your Liferay DXP instance to connect with
-    Liferay Analytics Cloud (see above).
+3.  Click *Add Data Source*. The *Add Data Source* page appears. 
 
-2.  Select *Settings* in the navigation panel.
+4.  Select the *Liferay DXP* icon. The *Configure Liferay DXP* page appears. The
+    Authorization tab (selected by default) shows data source and client
+    credentials fields. 
 
-3.  Click *Data Sources*. A listing of your existing data sources appears.
+    *Description* 
 
-4.  Click *Add Data Sources*. The *Connect Data Source* page appears. 
+    - *Name:* A name for your data source.
+    - *URL:* The Liferay DXP instance URL. 
 
-5.  Select the *Liferay DXP* icon. The *Connect to Instance* page appears and
-    lists your available Liferay DXP instances. 
+    *Client Credentials*
 
-6.  Select the instance to connect. The Liferay DXP Data Sources page appears, 
-    showing the instance description fields and configuration options. 
+    - *Consumer Key/Client ID:* Key/ID for Analytics Cloud to access your Liferay DXP instance. 
+    - *Consumer Secret/Client Secret:* Secret for Analytics Cloud to access your
+    Liferay DXP instance. 
 
-    *Description*
+    In Liferay DXP 7.1, you can find the Client ID and Secret by navigating to 
+    *Control Panel* &rarr; *Configuration* &rarr; *OAuth 2 Admin*.
 
-    - *URL:* The Liferay DXP instance URL.
-    - *Name:* A name for your data source. 
+    In Liferay DXP 7.0, you can find the Consumer Key and Secret on the *OAuth
+    Admin* page of the *Control Panel*. 
 
-    *Configuration Options*
+5.  Click *Authorize*. A window appears and asks if you want to authorize 
+    Analytics Cloud. 
 
-    - *Configure All:* Launches a wizard for configuring assets, touchpoints, 
-    and contacts from the instance. 
-    - *Configure Contacts:* Configures the contact data only.
-    - *Configure Analytics:* Configures the assets and touchpoints only.
+6.  Click *Authorize*. You've advanced to the *Configure Data Source* tab. 
 
-7.  Name your data source.
+Here are the configuration options:
 
-8.  Click *Configure All*. The Liferay DXP data source wizard appears, showing
+- *Configure All:* Launches a wizard for configuring assets, touchpoints, 
+and contacts from the instance. 
+- *Configure Contacts:* Configures the contact data only.
```

Comment:
I will send you screenshots for everything below here. It is mostly the same except there might be a few slight variations in the screenshots and UI

Reply:
Awesome! They've been very helpful!!